matrix_client/api.py

```diff
@@ -377,19 +377,15 @@ def media_upload(self, content, content_type):
 
     def get_display_name(self, user_id):
         content = self._send("GET", "/profile/%s/displayname" % user_id)
-        if "displayname" not in content.keys():
-            return user_id
-        return content['displayname']
+        return content.get('displayname', user_id)
```

Comment:
I'd prefer it if you return None here. It's perfectly okay to do it as a helper method, but there are times when I want to know if a displayname is set or not.